content/6_Advanced/Springboards.mdx

```diff
@@ -13,7 +13,15 @@ frequency: 1
 export const problems = {
 	sample: [new Problem('Gold', 'Springboards', '995', 'Hard', false, [], '')],
 	problems: [
-		new Problem('CSES', 'Projects', '1140', 'Easy', false, [], ''),
+		new Problem(
+		        'CSES',
+			'Projects',
+			'1140',
+			'Easy',
+			false,
+			[],
+			'cses-1140'
```

Comment:
> 'cses-1140'

where is the file?

Reply:
I think we can simply add the external link to the solution, which is this: https://codeforces.com/blog/entry/70018?mobile=false